source/extensions/omni.isaac.lab/omni/isaac/lab/controllers/operational_space.py

```diff
@@ -185,43 +134,83 @@ def reset_idx(self, robot_ids: torch.Tensor = None):
         """Reset the internals."""
         pass
 
-    def set_command(self, command: torch.Tensor):
+    def set_command(self, command: torch.Tensor, ee_pose: torch.Tensor | None = None):
         """Set target end-effector pose or force command.
 
         Args:
```

Comment:
please add ee_pose to args in docstring with shape info

Reply:
might also be good to add `current` or similar to the variable name so that it doesn't get confused with target

Reply:
I have changed it to current_ee_pose (please see comment above).